src/features/settings/SettingsControls/PlayerSetting/PlayerSetting.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react'
+import '../settings.css'
+import { PlayerGender, PlayerParts } from '../../../gameboard/types'
+import { useGA } from '../useGA'
+
+interface IPlayerSettingProps {
+  gender: PlayerGender
+  parts: PlayerParts
+  setParts: (newMode: PlayerParts) => void
+  setGender: (newGender: PlayerGender) => void
+}
+
+const PLAYER_GENDER_TYPES = [
+  { gender: PlayerGender.Female, name: 'Female', description: 'Refer to you as a female.' },
+  { gender: PlayerGender.Male, name: 'Male', description: 'Refer to you as a male.' },
+  { gender: PlayerGender.Neutral, name: 'Neutral', description: 'Refer to you neutrally.' },
```

Comment:
Nit pick: I think it might be good to also refer to the pronouns used here, just since `pup` might be a bit of a surprised

```suggestion
  { gender: PlayerGender.Female, name: 'Female', description: 'Refer to you as a female. (she/her/girl)' },
  { gender: PlayerGender.Male, name: 'Male', description: 'Refer to you as a male. (he/him/boy)' },
  { gender: PlayerGender.Neutral, name: 'Neutral', description: 'Refer to you neutrally. (pup)' },
```